backend/src/main/resources/application.properties

```diff
@@ -1,2 +1,44 @@
 spring.jmx.enabled=false
-server.port=8081
\ No newline at end of file
+server.port=8080
```

Comment:
8080 is al in gebruik op mijn machine, kunnen we 8081 of een andere poort gebruiken?